persistence/pom.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>com.derpgroup.derpwizard</groupId>
+		<artifactId>derpwizard-project</artifactId>
+		<version>0.0.2-SNAPSHOT</version>
```

Comment:
Actually, can't we just auto infer the version of the parent? We don't need to explicitly declare it here